modules/sentry/src/sentry.test.js

```diff
@@ -132,18 +132,41 @@ describe("Sentry.forwardLog", () => {
 });
 
 describe("Sentry.getUserUUID", () => {
-    test("When there is no Sentry.userUUID, sentry.getUserUUID return a different UUID each times it's called", () => {
+    test("When there is no user uuid in the local storage, Sentry.getUserUUID generate one and put it in userUUID", () => {
+        // To be sure there is no userUUID stored
+        try {
+            window.localStorage.sentryUUID = undefined;
+        } catch (e) {
+            console.warn("Local sotrage disabled");
+            return;
```

Comment:
During unit tests, we should not depend on localStorage availability
We should force its state by mocking it and write two tests
* a test with window.localStorage available (eg. window.localStorage = {})
* a test with window.localStorage unavailable (eg. window.localStorage = undefined)